tests/Server.php

```diff
@@ -40,6 +42,10 @@ public function setResponseBody(string $text, int $statusCode = 200)
 
     public static function boot()
     {
+        if (empty(getenv(self::ENV_SERVER_PORT))) {
```

Comment:
@sixlive 
I've added a check here to ensure that the env var is set before booting the server :smile: